pkg/cmd/root.go

```diff
@@ -60,7 +60,7 @@ func init() {
 	rootCmd.Version = version.ProgramVersion
 	rootCmd.SetVersionTemplate(rootCmd.Version + "\n")
 
-	rootCmd.PersistentFlags().String("key", "", "doppler api key")
+	rootCmd.PersistentFlags().StringP("key", "k", "", "doppler api key")
```

Comment:
if we are moving to using the word token instead of key, should we switch the "k" to "t"?

Reply:
Sure, I'll switch over to using `token` throughout. We'll have to manually update our existing config files. Alternatively, you can delete your config and let the migration script run again.

Reply:
Dope I will do that.